Fix postgres installation

mattboehm opened the pull request:
* rpm url needed to be updated to new version
* https://postgres.org now uses an SSL cert signed by letsencrypt, which
  is not trusted by default on centos6. Added steps to install that cert.

mattboehm committed:
Fix postgres installation

* rpm url needed to be updated to new version
* https://postgres.org now uses an SSL cert signed by letsencrypt, which
  is not trusted by default on centos6. Added steps to install that cert.

JeffreyMFarley commented on provisioners/roles/postgres/tasks/main.yml:
```diff
@@ -5,8 +5,28 @@
     dest=/etc/yum.repos.d/CentOS-Base.repo
   sudo: yes
 
+- name: install cert utilities
```
This might be the only 1 of the 4 new tasks you need.  It worked for me with recent yum/https issues

mattboehm replied:
So installing ca-certificates on a fresh vm and jumping straight to the "ensure PGDG Yum repo is installed" task works for you? I will try testing this when I reach a good stopping point.

mattboehm replied:
@JeffreyMFarley you're right; I've removed the unnecessary steps in d3d70b8

mattboehm committed:
Remove unnecessary postgres steps

Installing the cert utilities is all that's needed for the letsencryt
cert to be trusted
